src/domains/movies.js

```diff
@@ -0,0 +1,85 @@
+const { movie } = require('../utils/prisma')
+
+const getAllMoviesDb = async (runtimeLt, runtimeGt) => {
+  const movies = await movie.findMany({
+    where: {
+      runtimeMins: {
+        ...(runtimeLt ? { lt: Number(runtimeLt) } : {}),
+        ...(runtimeGt ? { gt: Number(runtimeGt) } : {})
+      }
+    },
+    include: {
+      screenings: true
+    }
+  })
+
+  return movies
+}
+
+const createMovieDb = async (title, runtimeMins, screenings) => {
+  const createdMovie = await movie.create({
+    data: {
+      title: title,
+      runtimeMins: runtimeMins,
+      ...(screenings && {
+        screenings: {
+          createMany: {
+            data: screenings
+          }
+        }
+      })
+    },
+    include: {
+      screenings: true
+    }
+  })
+
+  return createdMovie
+}
+
+const getMovieByIdDb = async (movieId) => {
+  const foundMovie = await movie.findFirst({
```

Comment:
I think [`findFirstOrThrow`](https://www.prisma.io/docs/orm/reference/prisma-client-reference#findfirstorthrow) may fit your needs here? I see you are throwing if this doesn't find you any results.